charts/em-stitching/values.yaml

```
@@ -21,10 +21,15 @@ stitching:
         - AppInsightsInstrumentationKey
         - docmosis-access-key
         - microservicekey-em-stitching-api
+        - stitching-POSTGRES-DATABASE
+        - stitching-POSTGRES-HOST
+        - stitching-POSTGRES-PASS
+        - stitching-POSTGRES-PORT
+        - stitching-POSTGRES-USER
   environment:
-    SPRING_DATASOURCE_URL: jdbc:postgresql://{{ .Release.Name }}-{{ .Values.postgresql.nameOverride}}:5432/postgres?ssl=disable
-    SPRING_DATASOURCE_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
-    SPRING_DATASOURCE_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    SPRING_DATASOURCE_URL: jdbc:postgresql://em-stitching-postgres-db-aat.postgres.database.azure.com:5432/stitching
```

https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L6

```suggestion
    SPRING_DATASOURCE_URL: jdbc:postgresql://em-stitching-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com:5432/stitching
```